draft-song-dns-wireformat-http-00.xml

```diff
@@ -125,98 +126,106 @@
-        or XML <xref target="I-D.mohan-dns-query-xml"/> encoding for DNS data, 
-        wire-format data is wrapped with a HTTP header and transmitted on port 80 
-        or 443. This protocol is not designed with web developers or
-        advanced usage in application development in mind. It simply
-        serves as a sort of DNS VPN which dose not require extra
-        parsing of another format of DNS data.
+    <t>Unlike a REST DNS API using JSON <xref target="I-D.bortzmeyer-dns-json"/>
+        or XML <xref target="I-D.mohan-dns-query-xml"/> encoding for
+        DNS data, in this approach wire-format data is wrapped with a
+        HTTP header and transmitted on port 80 or 443. This protocol
+        is not designed with web developers or advanced usage in
+        application development in mind. It simply serves as a sort of
+        DNS VPN, and does not introduce another format of DNS data.
        </t>
 
-      <t>This memo aims to introduce how the DNS binary over HTTP(s) concept works. 
-        Hopefully implementations by different developers following
-        this demo can speak with each other.</t>
+      <t>This memo aims to describe how the DNS binary over HTTP
+        concept works. Hopefully implementations by different
+        developers following this memo can speak with each other.
+      </t>
 
       <t>This mechanism is designed for client stub resolver to
-        recursive server. DNS zone transfer is out-of-scope for
-        this document.
+        recursive server. DNS zone transfer, DNS updates, and anything
+        other than simple DNS queries are out-of-scope for this
+        document.
       </t>
 
     </section>
 
     <section title="Methodology and Configuration">
 
-      <t>As it is mentioned in introduction, the basic methodology is wrapping the DNS 
-        wire-format data into HTTP(S) header and transmitted on port 80 or 443. However, 
-        there are two configuration scenarios for implementation.</t>
+      <t>As mentioned in introduction, the basic methodology is
+        wrapping the DNS wire-format data into an HTTP header and
+        transmitting on port 80 or 443. However, there are two
+        different scenarios for implementation.</t>
 
       <t>Scenario 1:</t>
 
-      <t>The DNS implementation handles DNS queries and responses via
-        both UDP and TCP on port 53, and HTTP(S) on port 80 or 443.
-        That means new functionality is added to existing 
-        DNS software as a built-in API. It works as follows:</t>
+      <t>The DNS server implementation handles DNS queries and
+        responses via both UDP and TCP on port 53, and HTTP on port 80
+        or 443. It works as follows:</t>
       <t><list style="numbers">
         <t>The client creates a DNS query message.</t>
-        <t>The client encapsulates the DNS message in a HTTP(S)
+        <t>The client encapsulates the DNS message in a HTTP
```

Comment:
I had two reasons for this:

1. As far as I know, there is no HTTPS protocol, although this is commonly used to describe HTTP over TLS. We could define exactly what this means (maybe it already is in some RFC that we can reference), but I don't think it's necessary for the definition of this protocol.
2. We were already asked to try to stop using non-encrypted protocols in the review draft. That is not practical here, since we want to support open resolvers. I added some text around the issue of HTTP vs. HTTPS as a separate section, and removed all references to HTTPS elsewhere. The idea is to limit the TLS vs. non-TLS discussion to a single point in the document, so people can focus on the rest of the protocol.